pkgs/applications/graphics/img-cat/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "imgcat";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "trashhalo";
+    repo = "imgcat";
+    rev = "v${version}";
+    sha256 = "0a8i0q2f109dljc9pcxvg2csaz2h04m0mdidjcwdzg6x6ly6skbg";
+  };
+
+  vendorSha256 = "03dg7xz9vd81wx0h0igsnl3n6x5lvxkjg4ynjrxxcz3dy65w7da2";
+
+  doCheck = false;
+
+  meta = with lib; {
```

Comment:
You should set `meta.platforms`.

Reply:
`meta.platforms` is now included, thanks!